src/ducks/settings/AccountsSettings.styl

```diff
@@ -18,10 +18,18 @@
     &__bank
         maxed-flex-basis 20%
 
+        +small-screen
+            maxed-flex-basis 33%
+
     &__number
         maxed-flex-basis 15%
         color var(--coolGrey)
 
+        +small-screen
+            td&
+                background-color red
```

Comment:
red ? Shouldn' we use pomegranate or something ? I have trouble seeing where we want to use a red background.